feat: add NG911 check for deprecation template (#224)
agrc/porter: .github/ISSUE_TEMPLATE/2-deprecate-sgid-dataset.md

## Are there service dependencies

- [ ] Does an application depend on this data?
- [ ] Are clients querying this data with the WebAPI?

> - [ ] Is this data used in the Next-Generation 911 Motorola Aware Map? (@eneemann on `2022/00/00`)

Reviewer: we're in 2023 now, might as well make this easier?

```suggestion
- [ ] Is this data used in the Next-Generation 911 Motorola Aware Map? (@eneemann on `2023/00/00`)
```

> - [ ] Does an application depend on this data?
> - [ ] Are clients querying this data with the WebAPI?
> - [ ] Is this data used in the Next-Generation 911 Motorola Aware Map? (@eneemann on `2023/00/00`)
>     - If so, notify Josh Nielson (josh.nielson at motorolasolutions.com)

Reviewer: ```suggestion
- [ ] Are clients querying this data with the WebAPI?
- [ ] Is this data used in the Next-Generation 911 Motorola Aware Map? (@eneemann on `2023/00/00`)
    - [ ] Notify Josh Nielson (josh.nielson at motorolasolutions.com) (@eneemann on `2023/00/00`)
- [ ] Does any other application depend on this data?
```
